language/Portuguese/strings.po

```diff
@@ -1040,1156 +1083,1182 @@ msgid ""
 "%s\n"
 " removed from favorites list."
 msgstr ""
+"%s\n"
+" removido da lista de favoritos."
 
 #: src/frontend/mame/ui/selgame.cpp:441
 msgid "Configure Machine"
-msgstr ""
+msgstr "Configurar Máquina"
 
 #: src/frontend/mame/ui/selgame.cpp:445 src/frontend/mame/ui/miscmenu.cpp:908
 msgid "Plugins"
-msgstr ""
+msgstr "Plugins"
 
 #: src/frontend/mame/ui/selgame.cpp:853
 #, c-format
 msgid "Romset\t%1$-.100s\n"
-msgstr ""
+msgstr "Romset\t%1$-.100s\n"
 
 #: src/frontend/mame/ui/selgame.cpp:854
 #, c-format
 msgid "Year\t%1$s\n"
-msgstr ""
+msgstr "Romset\t%1$-.100s\n"
```

Comment:
This appears to be a copy/paste error from the previous string.

Reply:
Yes, sorry about that.
Fixed.